src/openxr/extensions/xr_ext_hand_tracking_extension_wrapper.cpp

```diff
@@ -0,0 +1,270 @@
+
+XRAPI_ATTR XrResult XRAPI_CALL XRExtHandTrackingExtensionWrapper::xrCreateHandTrackerEXT(
+        XrSession session,
+        const XrHandTrackerCreateInfoEXT *createInfo,
+        XrHandTrackerEXT *handTracker) {
+    if (xrCreateHandTrackerEXT_ptr == nullptr) {
+        return XR_ERROR_HANDLE_INVALID;
+    }
+
+    return (*xrCreateHandTrackerEXT_ptr)(session, createInfo, handTracker);
+}
+
+PFN_xrDestroyHandTrackerEXT xrDestroyHandTrackerEXT_ptr = nullptr;
+
+XRAPI_ATTR XrResult XRAPI_CALL XRExtHandTrackingExtensionWrapper::xrDestroyHandTrackerEXT(
+        XrHandTrackerEXT handTracker) {
+    if (xrDestroyHandTrackerEXT_ptr == nullptr) {
+        return XR_ERROR_HANDLE_INVALID;
+    }
+
+    return (*xrDestroyHandTrackerEXT_ptr)(handTracker);
+}
+
+PFN_xrLocateHandJointsEXT xrLocateHandJointsEXT_ptr = nullptr;
+
+XRAPI_ATTR XrResult XRAPI_CALL XRExtHandTrackingExtensionWrapper::xrLocateHandJointsEXT(
+        XrHandTrackerEXT handTracker,
+        const XrHandJointsLocateInfoEXT *locateInfo,
+        XrHandJointLocationsEXT *locations) {
+    if (xrLocateHandJointsEXT_ptr == nullptr) {
+        return XR_ERROR_HANDLE_INVALID;
+    }
+
+    return (*xrLocateHandJointsEXT_ptr)(handTracker, locateInfo, locations);
+}
+
+XrResult XRExtHandTrackingExtensionWrapper::initialise_ext_hand_tracking_extension(XrInstance instance) {
+    XrResult result;
+
+    result = xrGetInstanceProcAddr(instance, "xrCreateHandTrackerEXT", (PFN_xrVoidFunction *)&xrCreateHandTrackerEXT_ptr);
+    if (result != XR_SUCCESS) {
+        return result;
+    }
+
+    result = xrGetInstanceProcAddr(instance, "xrDestroyHandTrackerEXT", (PFN_xrVoidFunction *)&xrDestroyHandTrackerEXT_ptr);
+    if (result != XR_SUCCESS) {
+        return result;
+    }
+
+    result = xrGetInstanceProcAddr(instance, "xrLocateHandJointsEXT", (PFN_xrVoidFunction *)&xrLocateHandJointsEXT_ptr);
+    if (result != XR_SUCCESS) {
+        return result;
+    }
+
+    return XR_SUCCESS;
+}
+
+bool XRExtHandTrackingExtensionWrapper::initialize_hand_tracking() {
+    XrResult result;
+
+    if (!hand_tracking_ext) {
+        return false;
+    }
+
+#ifdef DEBUG
+    Godot::print("OpenXR initialise hand tracking");
+#endif
+
+    XrSystemHandTrackingPropertiesEXT handTrackingSystemProperties = {
+            .type = XR_TYPE_SYSTEM_HAND_TRACKING_PROPERTIES_EXT,
+    };
+
+    XrSystemProperties systemProperties = {
+            .type = XR_TYPE_SYSTEM_PROPERTIES,
+            .next = &handTrackingSystemProperties,
+    };
+
+    result = xrGetSystemProperties(openxr_api->get_instance(), openxr_api->get_system_id(), &systemProperties);
+    if (!openxr_api->xr_result(result, "Failed to obtain hand tracking information")) {
+        return false;
+    }
+
+    if (!handTrackingSystemProperties.supportsHandTracking) {
+        // The system does not support hand tracking
+        printf("Hand tracking is not supported\n");
+        return false;
```

Comment:
Something I never cleaned up but this should be using `Godot::print`, nitpicking :)

Reply:
Done!